notification-service/src/helpers/generateEmail.ts

```diff
@@ -0,0 +1,178 @@
+export function generateTextEmail(eventName: string, contractAddress: string, link: string, network: string) {
+    return `We just detected an update of the event "${eventName}" from the contract "${contractAddress}" on the network "${network}". View more info on ${link}`;
+}
+
+export default function generateEmail(eventName: string, contractAddress: string, link: string, network: string) {
```

Comment:
I think for the first version it would've been enough to send text-only emails (with links) (:

Where did you get this html?

Reply:
The template is free to use and has a MIT License. 

You can find it here:;

Reply:
It's ok, I guess! Please link this repo in the code as a comment if we can't import this file directly

In the future, we may want to setup [vue + mjml](https://kevcodez.de/posts/2019-10-12-responsive-emails-vuejs-mjml/) for the email and preview it in storybook so we can do more complex formatting, while still working with the tools we used to work